### DISCUSSION TITLE
Add folders support

### DISCUSSION THREAD

**Discussion by nonua** [Chatterino/chatterino2#3882] [Ideas]
I just had that thought that folder could be a useful feature, you'd be able to group them at your wish, for example channels with similar interest or topics like a folder for dev streams, one for English streams, ...

Here's what it could look like with 3 different example states
- 2 closed folder, 1 opened with accent color
- An opened folder that has a live channel
- A closed folder that has unread messages

![folder](https://user-images.githubusercontent.com/77514325/181823409-b92639cc-b853-4d75-8c82-aab2fd1b9413.png)

**Comment by Mm2PL** [comment]
related https://github.com/Chatterino/chatterino2/issues/2753 https://github.com/Chatterino/chatterino2/issues/1919 https://github.com/Chatterino/chatterino2/issues/1484 https://github.com/Chatterino/chatterino2/pull/2600

**Reply by nonua** [reply]
My bad I guess, I only search "folder" in Discussions

**Reply by leon-richardt** [reply]
While these are related, they are all closed. Is there a consensus that this feature or something similar should not be implemented? I believe it would be very useful to have and don't agree with closing them (especially not #1484).

**Reply by pajlada** [reply]
I'm OK with folders, but I don't have an idea for how it would look like/how they would be usable. Something something drag tabs onto a folder to add them to a folder
